test/seldon/model/test-rabbit-models

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o nounset -o errexit -o pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+
+MODELS_DIR=$SCRIPT_DIR/../../models
+
+COMPUTE_NAMESPACE=seldon-regression-test
+
+RABBITMQ_USER=$(kubectl -n rabbitmq get secret rabbitmq-cluster-default-user -o jsonpath="{.data.username}" | base64 --decode)
+RABBITMQ_PASSWORD=$(kubectl -n rabbitmq get secret rabbitmq-cluster-default-user -o jsonpath="{.data.password}" | base64 --decode)
+EXPECTED_IRIS_RESULT_PAYLOAD='{"data":{"names":["t:0","t:1","t:2"],"ndarray":[[0.0006985194531162835,0.00366803903943666,0.995633441507447]]},"meta":{"requestPath":{"classifier":"seldonio/sklearnserver:1.14.1"}}}'
+EXPECTED_CIFAR10_RESULT_PAYLOAD='1.45e-08,1.25e-09,1.63e-07,1.15e-01,1.74e-07,6.19e-06,8.85e-01,6.07e-09,7.44e-08,4.73e-09,'
+
+echo ">>> Sending predict message to iris rabbitmq model input queue..."
+kubectl -n rabbitmq exec rabbitmq-cluster-server-0 -c rabbitmq -- \
+    curl -X POST "http://localhost:15672/api/exchanges/%2F/amq.default/publish" \
+    --user "${RABBITMQ_USER}:${RABBITMQ_PASSWORD}" \
+    -H "Content-type: application/json" \
+    -H "Accept: application/json" \
+    -d '{"properties":{"content_type":"application/json"},"routing_key":"iris-model-rabbitmq-input","payload":"{\"data\": {\"ndarray\": [[1,2,3,4]] }}","payload_encoding":"string"}'
+
+sleep 10 # I wish there was a better way to do this
```

Comment:
There is:
https://gist.github.com/fernandoacorreia/b4fa9ae88c67fa6759d271b743e96063

Reply:
The curl below to retrieve the message doesn't exit with an error if it finds no message, it just returns the string "null".  So removing the sleep will require building some more logic to check for that null, possibly using the `with_backoff`. (I did get the `with_backoff` into the code and tried it in a script, it  looks very useful).

Reply:
I found a way to do this by using `jq -e` so that `jq` returns an error response code if it can't find the expected elements in the json message